src/components/DateRangePickerInput.jsx

```diff
@@ -33,6 +34,7 @@ const propTypes = forbidExtraProps({
   startDateId: PropTypes.string,
   startDatePlaceholderText: PropTypes.string,
   screenReaderMessage: PropTypes.string,
+  ariaLabelledBy: validIdList,
```

Comment:
This component has a range that includes two `DateInput` components, so you'll likely need two different props for each input (something like `startDateAriaLabelledBy`)

Reply:
Great point. We're just using the `SingleDatePicker` so I missed this.

Reply:
okay, added this in